### DISCUSSION TITLE
tutorials/l2evpn/intro/

### DISCUSSION THREAD

**Discussion by giscus** [srl-labs/learn-srlinux#52] [comments]
# tutorials/l2evpn/intro/

Open documentation for open Network OS

https://learn.srlinux.dev/tutorials/l2evpn/intro/

**Comment by sogo4** [comment]
Hola. ¿Cuál es la contraseña de la máquina creada por el docker para introducir después de conectarse por ssh admin@clab-evpn01-leaf1?

**Comment by sogo4** [comment]
Hi. Which one is the password of the machine created by de docker to introduce after connect by ssh admin@clab-evpn01-leaf1?

**Reply by hellt** [reply]
Hi, it is NokiaSrl1!

**Comment by AdnanMostar** [comment]
Hi,could you please tell me what is a  username and password for Apline Linux servers?

**Reply by hellt** [reply]
you don't connect to them via ssh, you do `sudo docker exec -it <container-name> ash` to get shell access

**Reply by AdnanMostar** [reply]
Thank you Hellt,
I have found the solution in the meantime . We can login per ssh and also the way you mentioned  is possible as well.
